lib/geo_combine/ckan_metadata.rb

```diff
@@ -21,7 +21,7 @@ def geoblacklight_terms
         dc_identifier_s: @metadata['id'],
         dc_title_s: @metadata['title'],
         dc_rights_s: 'Public',
-        layer_geom_type_s: @metadata['type'],
+        layer_geom_type_s: 'Mixed',
```

Comment:
I think this should be `Not Specified` rather than `Mixed` based on slack discussion